src/apis/Style.res

```diff
@@ -17,6 +17,13 @@ external dp: float => size = "%identity"
 
 let pct = num => num->Js.Float.toString ++ "%"
 
+module AspectRatio = {
+  type t
+
+  external float: float => t = "%identity"
+  external string: string => t = "%identity"
+}
+
```

Comment:
I would prefer to not support this.
This will make all aspect ratio verbose and I don't see any value in this change.
What is the benefit of doing `aspectRatio={AspectRatio.string("16 / 9")}` when you can do `aspectRatio={16 /. 9}` ?
The string is unsafe by nature. @cknitt any feedback on this ?

Reply:
Agreed, I'd also prefer to keep just float.